src/core/api/alerts-manager/AlertsManager.tsx

```diff
@@ -5,29 +5,23 @@ import { Alert } from 'ui';
 import { core } from '..';
 
 const AlertsManager = () => {
-  const [alerts, setAlerts] = useState<string[]>([]);
+  const [alert, setAlert] = useState<string>();
```

Comment:
Remove <string> type def and add '' as init param - ts will know what type is it because of type inference mechanism